app/gateway/2.8.x/install-and-run/docker.md

```diff
@@ -147,6 +162,7 @@ docker run -d --name kong-gateway \
  -e "KONG_ADMIN_ERROR_LOG=/dev/stderr" \
  -e "KONG_ADMIN_LISTEN=0.0.0.0:8001" \
  -e "KONG_ADMIN_GUI_URL=http://localhost:8002" \
+ -e "KONG_LICENSE_DATA=${KONG_LICENSE_DATA}" \
```

Comment:
I _think_ you can provide `-e KONG_LICENSE_DATA` and it will read from the environment by default, ignoring it if there is no value available

Reply:
Yes, you're totally right. And I realize that you also commented this in an earlier conversation now, which I missed. Tested, works, updating the doc.